controls/togglebutton/visual-states.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Visual States
+page_title: .NET MAUI ToggleButton Documentation - Visual States
+description: Learn how to set the border color, border thickness and other in different Visual States of the Telerik ToggleButton for .NET MAUI.
+position: 9
+tags: .net maui, telerik .net maui, ui for .net maui, toggle, button, microsoft .net maui
+slug: togglebutton-visual-states
+---
+
+# .NET MAUI ToggleButton Visual States
+
+The ToggleButton provides the following `CommonStates` visual states described in the table below:
+
+| Visual States | Description |
+| ------------- | --------------- |
+| `Normal` | Applies when the button is in normal state |
+| `Pressed` | Applies when the button is pressed |
+| `MouseOver` | Applies when the mouse pointer hovers over the control - applicable for Desktop |
+| `Disabled` | Applies when the button is disabled |
+| `Toggled` | Applies when the button is disabled |
```

Comment:
"is toggled" instead of "is disabled"?

Reply:
Also all descriptions from here on have become "when the button is disabled", so they probably need to be revised.